tsdown.config.ts

```diff
@@ -1,10 +1,10 @@
-import { defineConfig } from "tsup";
+import { defineConfig } from "tsdown";
 
 export default defineConfig({
-	bundle: false,
 	clean: true,
 	dts: true,
 	entry: ["src/**/*.ts", "!src/**/*.test.*"],
 	format: ["esm"],
 	outDir: "lib",
+	unbundle: true,
```

Comment:
I believe a few of these are redundantly setting their defaults:

* [`clean`](https://tsdown.dev/reference/api/Interface.Options#clean)
* [`dts`](https://tsdown.dev/reference/api/Interface.Options#dts): because `types` is present in `package.json`
* [`format`](https://tsdown.dev/reference/api/Interface.Options#format): as `"esm"` is an alias of `"es"`